pages/guides/python/flask.en-US.mdx

```diff
@@ -5,4 +5,36 @@ ogImageSubtitle: Deploy Flask App in Zeabur
 
 # Deploy Flask
 
-This page is not completed yet. If you are willing to help Zeabur to complete this page, so that more developers can quickly deploy their projects, please submit a Pull Request to  [zeabur/zeabur-docs](https://github.com/zeabur/docs)。
+This article will teach you how to deploy your [Flask](https://flask.palletsprojects.com/) project on Zeabur.
+
+## Step1: Create Flask Project
+
+You can get the example project from [flask-template](https://github.com/zeabur/flask-template). The Flask example is quite simple. It has a single route that returns a string.
+
+```python 
+from flask import Flask
+import os
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return "Welcome, this is a Flask app deployed on Zeabur"
+
+if __name__ == '__main__':
+    app.run(debug=True, port=os.getenv("PORT", default=5000), host='0.0.0.0')
+```
+
+Please note that Zeabur uses environment variable `PORT` to determine which port to forward. You will therefore need to set this environment variable and use `os.getenv` or other ways to configure the port number in your application. By default, Zeabur will use port 5000.
```

Comment:
Port 5000 is for local development. For services deploying on Zeabur, we need to use the environment variable.

Reply:
Sorry for the mistake. The code is copied from [flask-template](https://github.com/zeabur/flask-template), should I remove the `default` option here?

Reply:
@jingfelix You don't need to remove the default label because it means "when there is no PORT environment variable, run service on port 5000.

However, for services deployed on Zeabur, it is guaranteed that PORT environment variable is exist, so we don't need to mention the default=5000 in the documentation 😃

Reply:
@yuaanlin Got it. Fixed in the latest commit.